cmd/relay/commands/server.go

```diff
@@ -48,7 +48,7 @@ func cmdStart(c *cli.Context) error {
 	rootservice := cfg.LoadRootsService(client)
 	claimservice := cfg.LoadClaimService(mt, rootservice, ks, acc)
 	idservice := cfg.LoadIdService(client, claimservice, storage)
-	nameservice := cfg.LoadNameService(idservice, claimservice, ks, acc, cfg.C.Domain, cfg.C.Namespace)
+	nameservice := cfg.LoadNameService(claimservice, ks, acc, cfg.C.Domain, cfg.C.Namespace)
```

Comment:
Why is the relay loading the config for a name service?

Reply:
right, removed